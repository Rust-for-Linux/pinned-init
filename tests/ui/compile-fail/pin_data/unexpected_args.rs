use pin_init::*;

#[pin_data(Bar)]
struct Foo {}

fn main() {}
