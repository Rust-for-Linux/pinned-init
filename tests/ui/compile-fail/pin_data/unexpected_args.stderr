error: Expected nothing or `PinnedDrop` as arguments to `#[pin_data]`.
 --> tests/ui/compile-fail/pin_data/unexpected_args.rs:3:12
  |
3 | #[pin_data(Bar)]
  |            ^^^
